web/courses_test.go

```diff
@@ -26,7 +26,7 @@ var allCourses = []*qf.Course{
 		Tag:              "Spring",
 		Provider:         "fake",
 		OrganizationID:   1,
-		OrganizationPath: "test",
+		OrganizationName: "test",
```

Comment:
Nitpicking: I would prefer also here to use more descriptive names than `test` and `test-1` and so on. Suggest to use dat or qf prefixes…

Reply:
Changed the first organization that is used in all tests to `qfTestOrg`, the rest are only used in combination with the predefined set of test courses, so I named them based on the course code for these courses. Does it sound reasonable?